test/go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/networkservicemesh/networkservicemesh/test v0.0.0-00010101000000-000000000000
 	github.com/networkservicemesh/networkservicemesh/utils v0.3.0
 	github.com/onsi/gomega v1.10.3
+	github.com/packethost/packngo v0.5.1 // indirect
```

Comment:
was this something you actually added or something added by your IDE?

Reply:
My IDE added it. Should I remove it?

Reply:
yes and also revert the diffs to go.sum

Reply:
Sure will do!